Source/WebKit/NetworkProcess/DatabaseUtilities.cpp

```diff
@@ -76,6 +76,14 @@ ScopeExit<Function<void()>> DatabaseUtilities::beginTransactionIfNecessary()
     } });
 }
 
+static bool validSQLiteStatement(Expected<WebCore::SQLiteStatement, int> preparedStatement, int expectedErrorCode)
```

Comment:
This name is confusing since it calls `step()` which evaluates/run the statement on the database.

Maybe we should call this: `evaluateStatement()` or `evaluateStatementAndExpect()` since we pass in the expected result code. By the way, `expectedResultCode` may be more accurate than `expectedErrorCode` since I don't think `SQLITE_DONE` or `SQLITE_ROW` are error codes.